.mailmap

```diff
@@ -84,6 +84,7 @@ Jarrod Millman <millman@berkeley.edu> Jarrod Millman <jarrod.millman@gmail.com>
 Jay Bourque <jay.bourque@continuum.io> jayvius <jay.bourque@continuum.io>
 Jerome Kelleher <jerome.kelleher@ed.ac.uk> jeromekelleher <jerome.kelleher@ed.ac.uk>
 Johannes Schönberger <hannesschoenberger@gmail.com> Johannes Schönberger <jschoenberger@demuc.de>
+Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com> Joseph Fox-Rabinovitz <joseph.r.fox-rabinovitz@nasa.gov>
```

Comment:
I am trying to get rid of the second line in the current output of `git log --format="%aN <%aE>" | sort -u`:

    Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com>
    Joseph Fox-Rabinovitz <joseph.r.fox-rabinovitz@nasa.gov>

I did not notice that there was also this line:

    Mad Physicist <madphysicist@users.noreply.github.com>